src/dispatching/dispatcher.rs

```diff
@@ -214,7 +237,7 @@ impl<R, Err, Key> Dispatcher<R, Err, Key>
 where
     R: Requester + Clone + Send + Sync + 'static,
     Err: Send + Sync + 'static,
-    Key: Hash + Eq,
+    Key: Hash + Eq + Clone,
```

Comment:
This is a breaking change BTW. Sadly, without something like `HashMap::drain_filter` (unstable) we can't remove something from the map without coping keys.

Reply:
Add to `CHANGELOG.md`?